Setup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+y | sudo apt-add-repository "deb http://archive.ubuntu.com/ubuntu focal main universe"
+sudo apt-get update
+sudo apt-get install build-essential retry g++-7 ninja-build libvulkan1 python3 python3-dev python3-pip libpng-dev libtiff5-dev libjpeg-dev tzdata sed curl libtool rsync libxml2-dev git git-lfs
+
+pip3 install --user numpy
+pip3 install --user -Iv setuptools==47.3.1
+pip3 install --user distro
+pip3 install --user wheel auditwheel
+
+git clone -b ue5-dev https://github.com/carla-simulator/carla.git carlaUE5
+cd carlaUE5
+
+curl -L -O https://github.com/Kitware/CMake/releases/download/v3.28.3/cmake-3.28.3-linux-x86_64.tar.gz
```

Comment:
Think about a better way to ensure minimum cmake verison.

### Benefits of current implementation
- User no need to care about cmake version.
- User can change cmake version if desired.
- We are not changing global bash environment.

### Critics of this implementation
- cmake binary is non-git-versioned so `git clean -fdx` will remove it forcing to re-setup.

### Alternatives
- Upload cmake binary to carla repo.
- Add an external folder for cmake command like UnrealEngine5_carla one.